blueocean-dashboard/src/main/js/util/UpdateGetParam.js

```diff
@@ -25,3 +25,10 @@ export default function updateGetParamm(paramName, newParamValue, getParams) {
 
     return updatedParamsStr.slice(0, -1);
 }
+
+/*
+ * @deprecated
+ */
+export function updateGetParamm(paramName, newParamValue, getParams) {
```

Comment:
I hope this works... It's of course a shim for the api change

Reply:
these classes are not used outside of this project, so _shouldn't_ be a problem

Reply:
maybe grep through the code for `updateGetParamm` and then bash someone on the head who actually wrote that code ;)